src/to_ascii.cpp

```diff
@@ -20,13 +32,12 @@ bool constexpr is_ascii(std::u32string_view view) {
   return true;
 }
 
-bool constexpr is_ascii(std::string_view view) {
+constexpr bool is_ascii(std::string_view view) noexcept {
```

Comment:
I don't think a table is needed. You could so...

```c++
bool is_ascii(std::string_view view) noexcept {
    uint8_t sum = 0;
    for(uint8_t c : view) { sum ^= c; }
    return sum & 0x80;
 }
```
It is should be quite fast!!!!

The compiler ought to be able to autovectorize this (so we get SIMD for free).

Note that I recommend against doing the following...

```c++
bool is_ascii(std::string_view view) noexcept {
   return std::accumulate(view.begin(), view.end(), 0, 
          [] (uint8_t sum, uint8_t c) { return sum ^ c; }) & 0x80;
 }
```

It is more complicated and GCC might generate worse code.